u3-views/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const expHbs = require('express-handlebars')
+const { courseDetails, courseUnits, courseStudents } = require('./features/courseController')
+const { index } = require('./features/indexController')
+const { videoplayer, videoHome } = require('./features/videoController')
```

Comment:
ควรใช้ `videoPlayer` ดีกว่า (ควรตั้งชื่อตัวแปรด้วย `lowerCamelCase`)
Prefer `videoPlayer` (variables should be named with `lowerCamelCase`)